### ISSUE TITLE
Update examples to provide better guidance on required parameter for CVS-Software

### ISSUE THREAD

**Issue by okrause** [NetApp/terraform-provider-netapp-gcp#24]
Multiple users ran into issues recently when using the provider to provision CVS-Software. We should update the examples to make it easier to get it right. I will give it a short the next days.

For reference:
CVS-Software regions require specification of:

```tf
storage_class = "software"
zone = "<your_CVS_zone"
```

**Comment by lonico** [comment]
Also when is snapshot_policy required when creating a volume.   Is it specific to CVS-software, or a general requirement?  Or should the API not require it.

**Comment by okrause** [comment]
The API doesn't require snapshot_policy. Here the relevant part of the [swagger definition](https://cloudvolumesgcp-api.netapp.com/swagger.json):

```
 "Volume_v2": {
...
        {
          "required": [
            "creationToken",
            "protocolTypes",
            "network"
          ],
...
```

CVS-Performance works fine without it. API for CVS-Software accepts a volume creation without it, but eventually fails at the end of the volume creation. To me, this looks like a regression "CVS-Software versus CVS-Performance" of the machinery behind the API.

**Comment by okrause** [comment]
Just submitted PR #26 to update the volume create examples.

For now, I also added

```tf
  snapshot_policy {
    enabled = true
  }

```
to the examples so users don't run into the issue using the example code.

**Comment by okrause** [comment]
PR #26 was merged in 73dba8f . Closing.
